src/safe_apps/tests/test_models.py

```diff
@@ -16,9 +16,14 @@ class IconTestCase(TestCase):
     def test_icon_upload_path(self) -> None:
         safe_app = SafeAppFactory.create()
 
-        self.assertEqual(
-            safe_app.icon_url.url, f"/media/safe_apps/{safe_app.app_id}/icon.jpg"
+        path_regex = "|".join(
+            [
+                r"\/media\/safe_apps\/",
+                r"[0-9(a-f|A-F)]{8}-[0-9(a-f|A-F)]{4}-4[0-9(a-f|A-F)]{3}-[89ab][0-9(a-f|A-F)]{3}-[0-9(a-f|A-F)]{12}",
```

Comment:
This [allows parentheses and pipes](https://gist.github.com/johnelliott/cf77003f72f889abbc3f32785fa3df8d?permalink_comment_id=4376611#gistcomment-4376611) but I don't think we need worry as it's a test.

Reply:
Yes, I agree, I think it doesn't need to be super strict. I was thinking in even using `/\w{8}-\w{4}-\w{4}-\w{4}-\w{12}/` which is simpler and also more permissive. But I also think it's OK as this is a test. Thanks!